src/plots/Mesh/MeshAttributes.code

```diff
@@ -189,3 +189,103 @@ MeshAttributes::ProcessOldVersions(DataNode *parentNode,
             searchNode->RemoveNode("lineStyle");
     }
 }
+
+Target: xml2info
+Function: MeshViewerEnginePluginInfo::InitializePlotAtts
+Declaration: virtual void InitializePlotAtts(AttributeSubject *atts, const avtPlotMetaData &);
+Definition:
+// ****************************************************************************
+//  Method: MeshViewerEnginePluginInfo::InitializePlotAtts
+//
+//  Purpose:
+//    Initialize the plot attributes to the default attributes.
+//
+//  Arguments:
+//    atts      The attribute subject to initialize.
+//    md        The metadata used to initialize.
+//    atts      The variable name used to initialize.
+//
+//  Programmer: miller86 -- generated by xml2info
+//  Creation:   Mark C. Miller, Thu May 30 22:41:29 PDT 2019
+// ****************************************************************************
+static int colorIndex = 0;
+// Grabbed 29May19 from https://sashat.me/2017/01/11/list-of-20-simple-distinct-colors/
+static unsigned char SashaTrubetskoy22SimpleDistinctColors[][4] = {
+    {230,  25,  75, 255}, { 60, 180,  75, 255}, {255, 225,  25, 255}, {  0, 130, 200, 255}, {245, 130,  48, 255},
+    {145,  30, 180, 255}, { 70, 240, 240, 255}, {240,  50, 230, 255}, {210, 245,  60, 255}, {250, 190, 190, 255},
+    {  0, 128, 128, 255}, {230, 190, 255, 255}, {170, 110,  40, 255}, {255, 250, 200, 255}, {128,   0,   0, 255},
+    {170, 255, 195, 255}, {128, 128,   0, 255}, {255, 215, 180, 255}, {  0,   0, 128, 255}, {128, 128, 128, 255},
+    {255, 255, 255, 255}, {  0,   0,   0, 255}};
```

Comment:
Would it make sense to create a Discrete color table with these values and use it instead.  It might make a good alternative for Contour plots and others that make use of Discrete color tables.

Reply:
Thats a good idea.

Would I then change to this color map when *random* mode is selected and then change back to the old colormap after *random* is switched to something else?